keyboards/kyria/keymaps/j-inc/keymap.c

```diff
@@ -268,21 +268,21 @@ static void render_anim(void) {
          }
     }
     if(get_current_wpm() != 000) {
+        oled_on(); // not essentiall but turns on Master OLED with any alpha keypress
         if(timer_elapsed(anim_timer) > ANIM_FRAME_DURATION) {
             anim_timer = timer_read();
             animation_phase();
         }
         anim_sleep = timer_read();
     } else {
-        if(oled_on()) {
+        if(timer_elapsed(anim_sleep) > OLED_TIMEOUT) {
+            oled_off();
+        } else {
             if(timer_elapsed(anim_timer) > ANIM_FRAME_DURATION) {
                 anim_timer = timer_read();
                 animation_phase();
```

Comment:
Here is where the animation side timeout fails. What's strange is at a low value like 10000 it will function correctly, but at a a medium value like 60000 it will turn on and off periodically (if left on idle) but at a high value like 120000 it fails to turn off at all.

Reply:
Change the timer stuff to 32 bit, eg `uint32_t`, `timer_read32`, `timer_elapsed32`, etc.

Reply:
That solved it — fixes reflected in latest commit.

Reply:
Fixed! Thank you.